### ISSUE TITLE
Unable to use BIGINT

### ISSUE THREAD

**Issue by odino** [taozhi8833998/node-sql-parser#227]
On 1.7.4.

```
CREATE TABLE `sql_parser` (
  `id` bigint(20) NOT NULL AUTO_INCREMENT,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8
```

```
SyntaxError: Expected "#", "--", ".", "/*", "CHAR", "DATE", "DECIMAL", "INT", "INTEGER", "JSON", "NUMERIC", "SMALLINT", "TIME", "TIMESTAMP", "TINYINT", "VARCHAR", or [ \t\n\r] but "b" found.
```

**Comment by taozhi8833998** [comment]
Merge ur pr! I will publish a new version when I fix another issue #229.

**Comment by odino** [comment]
That would be great :) thanks!

**Comment by taozhi8833998** [comment]
I have published a new version of `1.7.5`,  you could give it a shot. If all things go well, close the issue when you're free. BYW, another issue has been closed by GitHub when pr merged! Thanks for your pr again.

**Comment by odino** [comment]
Works fine!
